### PR TITLE
Judicial system/AcceptAlternativeTravelBan in judge view

### PR DESCRIPTION
**Pull request by oddsson**
# Add the ability for judges to accept a claim with alternative travel ban.

https://app.asana.com/0/322488613152836/1199387306535844

## What

See ☝️

## Why

Registrars often need to handle this case in a special manner with a workaround that we are now eliminating.

## Screenshots / Gifs

screencapture-localhost-4200-domari-krafa-urskurdur-dced7c56-4eac-43a4-bb9a-8d45bd1af794-2021-01-07-15_20_17

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx [hunk 1]
```diff
@@ -145,91 +148,134 @@ export const RulingStepOne: React.FC = () => {
...
               <PoliceRequestAccordionItem workingCase={workingCase} />
             </Accordion>
           </Box>
-          <Box component="section" marginBottom={8}>
-            <Box marginBottom={2}>
+          <Box component="section" marginBottom={5}>
+            <Box marginBottom={3}>
               <Text as="h3" variant="h3">
-                Niðurstaða úrskurðar
+                Úrskurðar{' '}
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx:168]:
```suggestion
                Úrskurður{' '}
```

File: apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx [hunk 2]
```diff
@@ -132,7 +135,7 @@ export const RulingStepOne: React.FC = () => {
     }
 
     if (workingCase) {
-      setIsStepIllegal(isNextDisabled(requiredFields))
+      setIsStepIllegal(isNextDisabled(requiredFields) || !workingCase.decision)
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx:138]: Can radio buttons not be part of required fields?

File: apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx [hunk 3]
```diff
@@ -145,91 +148,134 @@ export const RulingStepOne: React.FC = () => {
...
+              textarea
+              required
+            />
           </Box>
           {workingCase.decision !== CaseDecision.REJECTING && (
-            <Box component="section" marginBottom={8}>
+            <Box component="section" marginBottom={7}>
               <Box marginBottom={2}>
                 <Text as="h3" variant="h3">
                   Gæsluvarðhaldstími
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx:281]: Samkvæmt hönnun ætti þetta að vera "Gæsluvarðhald" ef krafa samþykkt og "Farbann" ef úrskurðað í farbann.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx (hunk 2 above)
Comment (oddsson) [reply apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx:138]: That would require us to refactor the validation. The validation uses regex to validate values and in this case we are checking if `workingCase.decision` is `null` and I don't know how to write a regex that checks for nulls.

**Review by gudjong — APPROVED** [review]

**Pull request merged**